zulipterminal/ui_tools/boxes.py

```diff
@@ -525,17 +525,26 @@ def main_view(self) -> Any:
         self.text_box = ReadlineEdit(u"Search: ")
         # Add some text so that when packing,
         # urwid doesn't hide the widget.
-        self.msg_narrow = urwid.Text("DONT HIDE")
+        self.conversation_focus = urwid.Text("DONT HIDE")
         w = urwid.Columns([
-            ('pack', self.msg_narrow),
+            ('pack', self.conversation_focus),
             ('pack', urwid.Text("  ")),
             self.text_box,
         ])
         self.w = urwid.LineBox(
             w, tlcorner=u'', tline=u'', lline=u'',
-            trcorner=u'', blcorner=u'─', rline=u'',
-            bline=u'─', brcorner=u'─')
-        return [self.w]
+            trcorner=u'', blcorner=u'', rline=u'',
+            bline=u'', brcorner=u'')
```

Comment:
Does this `LineBox` actually do anything?

Reply:
Ah, I see what you mean here. Yup. This does not seem to add anything. Will remove the `LineBox` in the next iteration.